README.rst

```diff
@@ -219,27 +205,27 @@ If you'd like to modify a component via a branch and point to that
 branch in your CESM sandbox, use the following procedure (again, using
 CAM as an example):
 
-#. Create a CAM branch. Since CAM originates from a subversion
-   repository, you will first need to create a branch in that
-   repository. Let's assume you have created this branch and called it
+#. Create a CAM branch. Let's assume you have created this branch and called it
    **my_branch**.
 
-#. Update **Externals.cfg** to point to your branch. You can replace the
-   **tag** entry with a **branch** entry, as follows::
+#. Update **.gitmodules** to point to a hash on your branch. You can replace the
+   **tag** entry with a **hash** entry, as follows, note that we have also changed the url to
+   point to a personal fork::
+
+  [submodule "cam"]
```

Comment:
I think this line needs two more spaces (comparing it to lines [230-235 of the older version](https://github.com/ESCOMP/CESM/blob/cesm3_0_alpha03b/README.rst?plain=1#L230-L235))